### ISSUE TITLE
UNITE 8 vs 9

### ISSUE THREAD

**Issue by slambrechts** [hildebra/lotus2#47]
Hi,

I see you use UNITE v8 as a database for ITS/fungi. When comparing results with the latest version of the unite database, v9, taxonomic identification results are very different for many OTUs, and less OTUs get an identification at species or genus level. Is that the reason you work with UNITE v8 in Lotus2?

Best,
Sam

**Comment by hildebra** [comment]
Hey Sam,
no the reason why we use Unite8 is that I don't really have much time to keep everything up to date. I would recommend U9 over U8, if you have it already. Will try to update at some point, but it's not on my high prio list as U8 is still a good DB in my experience.
best,
Falk

**Comment by luzhang321** [comment]
Hi :)
I notice that the current unite database for lotus2 has been updated to v9. I am wondering which type is exactly use in lotus2 since unite provides so many types for v9(https://unite.ut.ee/repository.php )?
For example, the dynamic version, 97 or 99? and includes only manual sigleton or includes the 'global and 3% distance singletons'? Includes all eukarytoes or only include fungi? 

Thanks so much!
Best,
Lu

**Comment by hildebra** [comment]
Hey Lu,
it obviously depends a bit on your usecase, but as general purpose v9 UNITE I added the "biggest" database, ie UNITE with all euks where available (https://doi.org/10.15156/BIO/2938069). This option was used to provide the most general purpose DB, using the dynamic singletons.
Would you have a specific preference for either DB version?
best,
Falk

**Comment by luzhang321** [comment]
Thank you very much for your prompt reply @hildebra. If I understand correctly, lotus2 now uses the general version UNITE with all eukarytoes with singletons set as RefS (in dynamic files). 
In my case, my aim is to identify fungi distributions of my data. But I accidently found that there exist some protista in my samples. That's why I ask which database should I used for ITS database. 
I checked some forums and literatures about unite database. 

- https://journals.asm.org/doi/full/10.1128/aem.01047-20 
In this paper they wrote 
"Given that assignment of taxonomy is critical in this approach that relies on species-level classification, we compared taxonomic identifications of the fungal amplicon sequence variants (ASVs) between two UNITE fungal reference databases that handle taxonomic singletons differently, **one including all global taxonomic singletons and another including only singletons manually curated by taxonomic experts, known as reference singletons** ([19](https://journals.asm.org/doi/full/10.1128/aem.01047-20#core-B19)). We refer to these two distributions of the reference databases as UNITE with global singletons and UNITE with reference singletons.
**The results when taxonomy was assigned with the UNITE reference singletons were, overall, similar to the taxonomic assignments made with the UNITE global singletons**, and a comparison between the two taxonomic assignments is included in File S1 in the supplemental material."
- https://forum.qiime2.org/t/unite-classifiers/21895
There is also a discussion in qimme2 forum. 
It is mentioned that both 99% and dynamic version works fine. "**Clustering does not make that much of an impact but I would discourage you from using the 97%... you do lose some taxonomic information. 99% or dynamic would be best. It is probably not worth the trouble of training multiple, as the differences are quite minor.**"
They has published a paper about this: 
https://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1009581
In 'Reference curation improves taxonomic classification: Lessons from the UNITE Fungal ITS database' part, 
**OTU clustering (97% vs. 99% vs. dynamic clustering) exhibited minimal impact on results, though the 99% OTUs yielded the highest taxonomic information and classification accuracy** ([Fig 9](https://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1009581#pcbi-1009581-g009)) & **Taxonomic classification accuracy is lower in the “all eukaryote” release version of the UNITE database, compared to fungal sequences alone** ([Fig 9D and 9E](https://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1009581#pcbi-1009581-g009)).
- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10714301/
Next, the Qiime classifier was trained using the **UNITE database (version 8.3, all eukaryotes)** with highest number of reference sequences (RefS) as compared to representative sequences (RepS)[[29](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10714301/#B29)].**Non-fungal kingdoms were removed** and consisted of Alveolata, Chromista, Eukaryota kgd Incertae sedis, Metazoa, Stramenopila, and Viridiplantae.
- https://onlinelibrary.wiley.com/doi/full/10.1111/mec.16460
In this paper, eukarytoes one is suggested. We recommend identification of **fungal and other eukaryote ITS sequences based on the UNITE reference data set**, because it is the largest curated database and it includes multiple nonfungal reads to facilitate separation of fungi from other eukaryotes (Anslan et al., [2018](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.16460#mec16460-bib-0017)).
- https://www.nature.com/articles/s41564-022-01147-3#Sec8
For ITS, the ASVs or OTUs were taxonomically annotated with RDP Classifier using **UNITE Fungal ITS training set** (version of August 2018)[52](https://www.nature.com/articles/s41564-022-01147-3#ref-CR52);
- https://journals.asm.org/doi/full/10.1128/aem.02825-19
To obtain taxonomic assignment of the reads post-quality control, the **dynamic UNITE fungal database** with a 97 to 99% similarity threshold was used to fit and train a naive-Bayes taxonomic classifier.
- https://github.com/benjjneb/dada2/issues/702
They wrote ‘As far as guidance, we do not have any on which to use at this time. They both work. **I usually use the non-singleton version, but not for any very good reason, just because its a bit smaller.** We have not any rigorous testing comparing the two.' 

It seems that there is not a good answer for using which database. I personally prefer the dynamic version with RefS, I think it is more conservative. The dynamic is 'some SHs are delimited at the 3% distance level, some at the 2.5% distance level, some at the 2% distance level, and so on; these choices were made manually by experts of those particular lineages of fungi.' 
And for the eukarytoes and fungi database, I perfer to use the eukarytoes (the protista result seeems reasonable), but probably keep fungi profile only. 

**Another small question** is the unite database in my previously used lotus 2.24 version is a qimme version with /home/lu/miniconda3/envs/lotus2/share/lotus2-2.24-0//DB//UNITE/sh_refs_qiime_ver8_99_s_all_02.02.2019.fasta . 
Is there any particular reason that it is has been changed now? 

Thanks a lot! 
Best,
Lu

**Comment by hildebra** [comment]
Hey Lu,
thanks for your good research into this topic. Overall I agree with the overall vibe of these different sources, I would reason similarly:
- use 99% OTUs to "catch" fine grained details
- use Euks, so you know that you have sequenced a protists - getting a false positive assignment to a Fungi won't help you, you really need to know that this is a false positive amplification.
- Benjamin is right that smaller databases will make your analysis run faster. However, I would argue that LotuS2 is so fast that is doesn't make a difference whether your computer spent 2-3 min longer, given the benefit of maybe catching an additional reference.
- I would remove the protists in a later step in R/Matlab (or where you analyse your data). The reason is: removing is no problem, having a false-positive fungi (that is actually a protist) is a problem!

In summary: for me the best database is 99%, including euks. Only downside : its a bit slower (no problem for LotuS2 though), upside: higher sensitivity, can detect false positive amplicons and you are sure you did use the most extensive ref data available.

No particular reason for changing from QIIME -> general fasta. Just thought this is more future proof. 
Btw, you can easily run a custom database in LotuS2 (e.g. download the UNITE version you prefer), following these instructions:
https://github.com/hildebra/lotus2?tab=readme-ov-file#coustom-reference-database
and
https://lotus2.earlham.ac.uk/main.php?site=documentation#usage (section refDB  and tax4refDB )

Hope this helps in your quest, and please let me know what you find doing such an extensive comparison of pipelines!
 best,
Falk
